tests/gpflow/utilities/test_deepcopy.py

```diff
@@ -1,17 +1,16 @@
-from copy import deepcopy
 import pytest
+import copy
 
 import tensorflow as tf
 import tensorflow_probability as tfp
-from gpflow.utilities import deepcopy_components
+from gpflow.utilities import deepcopy
 
 
 class A(tf.Module):
```

Comment:
`A` and `B` could do with more descriptive names - what are their purposes? what distinguishes them?